### ISSUE TITLE
parsing taxon name/gene_desc in ncbi_searcher

### ISSUE THREAD

**Issue by mpnelsen** [ropensci/traits#33]
Hi - I'm having a minor issue when using ncbi_searcher.  I've encountered an ncbi accession for which the taxon name and gene_desc are parsed in an undesired fashion.  I think perhaps the first two words of the definition are used as the taxon name, and the remainder goes to gene_desc...but this particular accession description begins 'UNVERIFIED: Genus species ...', so the output for taxon name is 'UNVERIFIED: Genus', and then the gene_desc starts with 'Species...'.  

require(traits)
search.res<-ncbi_searcher(id="48848",entrez_query="RPB1",seqrange="350:3500")
search.res

I'm having trouble tracking down where the issue lies in the code...any suggestions which (internal) functions I might check?  Thanks much!

-Matt.

**Comment by sckott** [comment]
hi @mpnelsen - good catch! 

I'll see if I can fix the parsing on this

**Comment by mpnelsen** [comment]
thanks much.  i tried tinkering with a couple of functions but was unsuccessful in tracking it down.

**Comment by sckott** [comment]
@mpnelsen okay, reinstall from github `devtools::install_github("ropensci/traits")`, then try that eg again. Should get 

```r
ncbi_searcher(id="48848",entrez_query="RPB1",seqrange="350:3500")
```

```r
                        taxon length
1 Agaricostilbum pulcherrimum    811
                                                                                                                       gene_desc   acc_no     gi_no
1 UNVERIFIED: Agaricostilbum pulcherrimum strain TUB 020329 RNA polymerase II largest subunit-like (RPB1) gene, partial sequence KF211359 514996267
```

**Comment by sckott** [comment]
the `gene_desc` now isn't parsed at all b/c it's too hard to parse the names off the front given the variation in that string, make a separate call to the taxonomy database to get the taxonomic names: https://github.com/ropensci/traits/blob/master/R/ncbi_searcher.R#L204-L218

**Comment by mpnelsen** [comment]
thanks very much!

**Comment by sckott** [comment]
np :) :+1:
